api/python/notebooks/analysis_demo/comp_bio_geneformer_prediction.ipynb

```diff
@@ -182,10 +182,11 @@
     "\n",
     "### Preparaing single-cell data\n",
     "\n",
-    "Let's load the test data. In preparation to use with Geneformer the following: \n",
-    "- Sets the index as the ENSEMBL ID and stores it in the `obs` column `\"ensembl_id\"`\n",
-    "- Adds read counts to the `obs` column `\"n_counts\"`\n",
-    "- Adds and ID column to be used for joining later in the  `obs` column `\"joinid\"`\n",
+    "Let's load the test data. In preparation to use with Geneformer to do the following: \n",
```

Comment:
`In preparation to use with Geneformer to do the following` does not scan - needs an edit